src/khoj/interface/web/base_config.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🦅</text></svg>">
+        <title>Khoj - Configure App</title>
+    	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@picocss/pico@1/css/pico.min.css">
+    </head>
+    <body class="khoj-configure">
+        <header>
+            <div>
+                <h1>Configure Khoj</h1>
```

Comment:
Should we use "Setup" instead of "Configure" in all the UI headings and description for the config page? The terms "Settings" and "Setup" seem more standard. Or maybe just say "Settings" (Khoj in the title is redundant, maybe)
 
```suggestion
                <h1>Settings</h1>
```